apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -393,41 +434,50 @@ const ApplicationReview = ({
         <Heading level="h3" data-h2-margin="base(0, 0, x1, 0)">
           {intl.formatMessage({
             defaultMessage: "Sign and submit",
-            id: "+sWqVY",
+            id: "fhgZRX",
+            description:
+              "Heading for sign and submit section of application review page.",
           })}
         </Heading>
         <div>
           <FormProvider {...methods}>
             <form onSubmit={methods.handleSubmit(handleSubmit)}>
               <p data-h2-margin-bottom="base(x1)">
                 {intl.formatMessage({
-                  defaultMessage: `"You made it! By signing your name, you confirm that:"`,
-                  id: "d0KeTK",
+                  defaultMessage: `You made it! By signing your name, you confirm that:`,
+                  id: "e05s3z",
+                  description:
+                    "Heading for sign and submit section of application review page.",
                 })}
               </p>
               <ul>
                 <li>
                   <p data-h2-margin-bottom="base(x.5)">
                     {intl.formatMessage({
                       defaultMessage: `"I’ve reviewed everything written in my application"`,
-                      id: "IFVdRD",
+                      id: "Th2Tu0",
+                      description:
+                        "List item for sign and submit section of application review page.",
```

Comment:
The same description is used for each list item. Consider using the following to differentiate and better describe:

- Review list item for sign and submit section of application review page.
- Community list item for sign and submit section of application review page.
- Truth list item for sign and submit section of application review page.